docs/src/main/sphinx/connector/kudu.rst

```diff
@@ -189,72 +189,102 @@ tables are mapped to schemas depending on some conventions.
-+-----------------------+-----------------------+-----------------------+
-| ``VARBINARY``         | ``BINARY``            | see [1]_              |
-+-----------------------+-----------------------+-----------------------+
-| ``TIMESTAMP``         | ``UNIXTIME_MICROS``   | µs resolution in Kudu |
-|                       |                       | column is reduced to  |
-|                       |                       | ms resolution         |
-+-----------------------+-----------------------+-----------------------+
-| ``DECIMAL``           | ``DECIMAL``           | only supported for    |
-|                       |                       | Kudu server >= 1.7.0  |
-+-----------------------+-----------------------+-----------------------+
-| ``CHAR``              | \-                    | not supported         |
-+-----------------------+-----------------------+-----------------------+
-| ``DATE``              | \-                    | not supported [2]_    |
-+-----------------------+-----------------------+-----------------------+
-| ``TIME``              | \-                    | not supported         |
-+-----------------------+-----------------------+-----------------------+
-| ``JSON``              | \-                    | not supported         |
-+-----------------------+-----------------------+-----------------------+
-| ``TIME WITH           | \-                    | not supported         |
-| TIMEZONE``            |                       |                       |
-+-----------------------+-----------------------+-----------------------+
-| ``TIMESTAMP WITH TIME | \-                    | not supported         |
-| ZONE``                |                       |                       |
-+-----------------------+-----------------------+-----------------------+
-| ``INTERVAL YEAR TO MO | \-                    | not supported         |
-| NTH``                 |                       |                       |
-+-----------------------+-----------------------+-----------------------+
-| ``INTERVAL DAY TO SEC | \-                    | not supported         |
-| OND``                 |                       |                       |
-+-----------------------+-----------------------+-----------------------+
-| ``ARRAY``             | \-                    | not supported         |
-+-----------------------+-----------------------+-----------------------+
-| ``MAP``               | \-                    | not supported         |
-+-----------------------+-----------------------+-----------------------+
-| ``IPADDRESS``         | \-                    | not supported         |
-+-----------------------+-----------------------+-----------------------+
-
-
-.. [1] On performing ``CREATE TABLE ... AS ...`` from a Trino table to Kudu,
-   the optional maximum length is lost
-
-.. [2] On performing ``CREATE TABLE ... AS ...`` from a Trino table to Kudu,
-   a ``DATE`` column is converted to ``STRING``
+Type mapping
```

Comment:
Add `.. _kudu-type-mapping:` anchor

Reply:
Oops. I've only been adding them if the section is referenced, since not all sections in our docs include anchors. Should I add that to the type-mapping punch list?

Reply:
Yes please do, it should be a standard like the SQL support section.